### PR TITLE
feat: add logging level per environment

### PR DESCRIPTION
**Pull request by pietro-tota**
#### List of Changes

- Parametrize logging level
- Lowered MdcFilter request logging level to debug level

#### Motivation and Context
Each received request is logged, also the Kubernetes liveness and readiness ones increasing module log verbosity

#### How Has This Been Tested?
Local test and dev test checking the module logs

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by giovanniberti — APPROVED** [review]

File: helm/values-uat.yaml [hunk 1]
```diff
@@ -52,6 +52,8 @@ microservice-chart:
       PAYMENT_TOKEN_VALIDITY_TIME: "1800"
       TRANSACTIONS_RETRY_OFFSET: "60"
       CLOSURE_RETRY_INTERVAL: "30"
+      WEB_LOGGING_LEVEL: "off"
+      PAGOPA_LOGGING_LEVEL: "info"
```
Comment (giovanniberti) [inline helm/values-uat.yaml:56]: What about `PAGOPA_LOGGING_LEVEL`?

**Review by giovanniberti — APPROVED** [review]

File: helm/values-uat.yaml (hunk 1 above)
Comment (pietro-tota) [reply helm/values-uat.yaml:56]: I put here pagopa referring the application package name 'it.pagopa...'. Maybe with this consideration in mind we can update it to `TRANSACTION_LOGGING_LEVEL` maybe

Comment (giovanniberti) [reply helm/values-uat.yaml:56]: Yeah even `TRANSACTIONS_LOGGING_LEVEL` would be ok. From an outsider point of view it should be understandable what each environment variable does, `PAGOPA_LOGGING_LEVEL` ties a little too much into an implementation detail (maybe even `SERVICE_LOGGING_LEVEL` or `BUSINESS_LOGGING_LEVEL` would be ok I guess)

Comment (pietro-tota) [reply helm/values-uat.yaml]: I like `SERVICE_LOGGING_LEVEL`.
Refactored to SERVICE_LOGGING_LEVEL into https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/119/commits/e31c870d5011dc8fe441642515c0b6d0c93daca9

**Review by infantesimone — APPROVED** [review]

**Pull request merged**